OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -801,6 +801,22 @@ internal static void UpgradeWeaponRules(ModData modData, int engineVersion, ref
 					}
 				}
 
+				// Replaced projectile.TargetCenterPosition with WeaponInfo.TargetActorCenter
+				if (engineVersion < 20170622)
+				{
+					var instantHit = node.Value.Nodes.FirstOrDefault(n => n.Key == "Projectile" && n.Value.Value == "InstantHit");
+					if (instantHit != null)
+					{
+						var targetCenterPos = instantHit.Value.Nodes.FirstOrDefault(n => n.Key == "TargetCenterPosition");
+						if (targetCenterPos != null)
+						{
+							Console.WriteLine("The projectile TargetCenterPosition property was replaced with weapon TargetActorCenter property.");
+							instantHit.Value.Nodes.Remove(targetCenterPos);
+							node.Value.Nodes.Add(new MiniYamlNode("TargetActorCenter", "true"));
```

Comment:
I thought InstantHit already had that property long before, but you're right, this can actually be dropped.